app/ErrorHandler.scala

```diff
@@ -48,26 +49,34 @@ class ErrorHandler @Inject()(
 
     if (statusCode == FORBIDDEN)
       Future.successful(Forbidden(standardErrorTemplate("global.error.403.title", "global.error.403.heading", "global.error.403.message")(request)))
-    else
+    else {
+      logger.error(s"onClientError $message")
       super.onClientError(request, statusCode, message)
+    }
   }
 
   override def resolveError(request: RequestHeader, exception: Throwable): Result = {
     auditServerError(request, exception)
 
     exception match {
-      case _: NoActiveSession =>
+      case e: NoActiveSession =>
+        logger.error(s"NoActiveSession ${e.getMessage}")
```

Comment:
why an error here?

Reply:
That is the ticket requirement.
https://jira.tools.tax.service.gov.uk/browse/APB-4996